src/components/toolbar/Toolbar.js

```diff
@@ -25,12 +25,19 @@ export default {
         'toolbar--fixed': this.fixed,
         'toolbar--floating': this.floating,
         'toolbar--prominent': this.prominent,
+        'toolbar--extended': this.isExtended(),
         'theme--dark': this.dark,
         'theme--light': this.light
       }
     }
   },
 
+  methods: {
```

Comment:
For similarity of how we do this in other places, move this to a computed property